tests/common-tests.h

```diff
@@ -596,7 +596,7 @@ TEST_CASE_METHOD(common_tests, "Use and into", "[core][into]")
         CHECK(sh == 3);
     }
 
-    SECTION("Round trip works for int")
+    SECTION("Round trip works for int", "section")
```

Comment:
Same here, I'd make it a proper tag like `["int"]`

**Unless**, we are going to get rid of it once SOCI switches to C++11 anyway.

If so, we may agree to replace all those with `["dummy"]` (or even empty string if kosher for Catch) as we don't seem to use the tags currently anyway. That would be simpler to re-work the PR with search & replace.

Either would work for me.